specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -5617,16 +5601,16 @@
         "platformFaultDomain": {
           "type": "integer",
           "format": "int32",
-          "description": "Fault domain of the host within a group. Supported values 0,1,2."
+          "description": "Fault domain of the dedicated host within a dedicated host group. Supported values 0,1,2."
```

Comment:
Could also use a min/max to describe the restriction

Reply:
What is the syntax for that?

Reply:
Found this in the repo
https://github.com/Azure/azure-rest-api-specs/blob/73cd215474d2f695ded625fb38f3def8d26a5779/specification/analysisservices/resource-manager/Microsoft.AnalysisServices/stable/2017-08-01/analysisservices.json#L962-L968

Reply:
Although adding that rather than just the description, may be a breaking change

Reply:
thanks! it is not breaking as service always rejected the requests with out of boundary values.